building/glass/branches-schema.yaml

```diff
@@ -0,0 +1,34 @@
+$schema: "http://json-schema.org/draft-07/schema#"
+title: BranchesConfig
+type: object
+properties:
+  branches:
+    type: array
+    items:
+      type: object
+      properties:
+        name:
+          type: string
+          pattern: "[a-z0-9.-]+/[a-z0-9-]+"
+        signing-key:
+          type: string
+          pattern: "[a-fA-F0-9]+"
+        apt-url-prefix:
+          type: string
+        upload:
+          type: object
+          oneOf:
+          - properties:
+              method:
+                enum: ["google-cloud-storage"]
+          - properties:
+              method:
+                enum: ["rsync"]
+              username:
+                type: string
+              host:
+                type: string
+              dir:
+                type: string
+      required: ["name", "signing-key"]
```

Comment:
Should probably disallow other unrecognized fields on all `type: object` schemas, so that typos are more likely to be caught.